src/etc/inc/captiveportal.inc

```diff
@@ -347,15 +347,10 @@ EOD;
 		@unlink("{$g['varetc_path']}/captiveportal-{$cpzone}-error.html");
 		@unlink("{$g['varetc_path']}/captiveportal-{$cpzone}-logout.html");
 
-		captiveportal_radius_stop_all();
+		captiveportal_disconnect_all(10); // NAS-Request
 
 		captiveportal_filterdns_configure();
 
-		/* send Accounting-Off to server */
-		if (!platform_booting()) {
-			captiveportal_send_server_accounting(true);
```

Comment:
@jim-p I found during testing that if many zones share a radius server and one of the zones is disabled, that function sends a stop message to the server and the server stops the accounting for all the clients it knows, including clients in other zones who are still getting service. 
At that point the RADIUS server loses track of the number of active sessions and their length and it can't enforce Simultaneous-Use limits or other limits any more.